### ISSUE TITLE
py2exe not found

### ISSUE THREAD

**Issue by daurnimator** [kennethreitz/legit#88]
```
$ pip install legit
Downloading/unpacking legit
  Downloading legit-0.1.1.tar.gz
  Running setup.py egg_info for package legit
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/tmp/pip-build/legit/setup.py", line 37
        print 'py2exe is required to continue.'
                                              ^
    SyntaxError: invalid syntax
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/tmp/pip-build/legit/setup.py", line 37

    print 'py2exe is required to continue.'

                                          ^

SyntaxError: invalid syntax

----------------------------------------
Command python setup.py egg_info failed with error code 1 in /tmp/pip-build/legit
Storing complete log in /home/daurnimator/.pip/pip.log
```

Running arch x86_64

**Comment by nblackburn** [comment]
+1

**Comment by MaikuMori** [comment]
Those one archlinux if you want to install using pip, then use `pip2 install legit`.

**Comment by hickford** [comment]
Try installing from develop:

    pip install git+https://github.com/kennethreitz/legit.git@develop

Does that install correctly? Otherwise please reopen the issue

**Comment by daurnimator** [comment]
> Try installing from develop:
> 
> pip install git+https://github.com/kennethreitz/legit.git@develop
> Does that install correctly? Otherwise please reopen the issue

No:

```
 $ pip install git+https://github.com/kennethreitz/legit.git@develop
DEPRECATION: --download-cache has been deprecated and will be removed in the future. Pip now automatically uses and configures its cache.
Collecting git+https://github.com/kennethreitz/legit.git@develop
  Cloning https://github.com/kennethreitz/legit.git (to develop) to /tmp/pip-n8h4uon2-build
Collecting clint>=0.4.1 (from legit==0.1.1)
  Downloading clint-0.4.1.tar.gz
Collecting GitPython>=0.3.4 (from legit==0.1.1)
  Downloading GitPython-1.0.1.tar.gz (355kB)
    100% |████████████████████████████████| 356kB 996kB/s 
Requirement already satisfied (use --upgrade to upgrade): six>=1.9.0 in /usr/lib/python3.4/site-packages (from legit==0.1.1)
Collecting args (from clint>=0.4.1->legit==0.1.1)
  Downloading args-0.1.0.tar.gz
Collecting gitdb>=0.6.4 (from GitPython>=0.3.4->legit==0.1.1)
  Downloading gitdb-0.6.4.tar.gz (400kB)
    100% |████████████████████████████████| 401kB 993kB/s 
Collecting smmap>=0.8.5 (from gitdb>=0.6.4->GitPython>=0.3.4->legit==0.1.1)
  Downloading smmap-0.9.0.tar.gz
Installing collected packages: args, clint, smmap, gitdb, GitPython, legit
  Running setup.py install for args
    Complete output from command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-8040ow33/args/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-73ixwz8b-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib
    copying args.py -> build/lib
    running egg_info
    writing args.egg-info/PKG-INFO
    writing top-level names to args.egg-info/top_level.txt
    writing dependency_links to args.egg-info/dependency_links.txt
    warning: manifest_maker: standard file '-c' not found
    
    reading manifest file 'args.egg-info/SOURCES.txt'
    writing manifest file 'args.egg-info/SOURCES.txt'
    running install_lib
    copying build/lib/args.py -> /usr/lib/python3.4/site-packages
    error: could not create '/usr/lib/python3.4/site-packages/args.py': Permission denied
    
    ----------------------------------------
Command "/usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-8040ow33/args/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-73ixwz8b-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-8040ow33/args
```

**Comment by hickford** [comment]
@daurnimator that's a general permission error, try `pip install --user git+https://github.com/kennethreitz/legit.git@develop` to install in your home folder (or use sudo)

**Comment by hickford** [comment]
Published v0.2.0 to PyPI https://pypi.python.org/pypi/legit/

**Comment by daurnimator** [comment]
Great, thanks!
Got it installed now, 2.5 years after I first saw the project.
